### PR TITLE
Fix case where user doesn't have Lambda access and tries to open a CloudFormation stack

### PR DESCRIPTION
**Pull request by awschristou**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description

Accounts without Lambda access throw an exception when trying to open a CloudFormation Stack that contains Lambda Functions.
Now the stack can be opened, and Lambda stack resources will have Error Nodes as a more graceful handling.

image

### CODE REVIEW

**Comment by abrooksv** [comment]
Can we give a better reason like we just did in #637

**Comment by awschristou** [comment]
@abrooksv when #637 gets submitted, this one automatically benefits from it

**Comment by abrooksv** [comment]
Ah. K

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt [hunk 1]
```diff
@@ -105,23 +106,29 @@ class CloudFormationStackNode(project: Project, val stackName: String, private v
...
-
-            LambdaFunctionNode(
-                nodeProject,
-                client,
-                response.configuration().toDataClass(credentialProvider.id, region),
-                true
-            )
+    private fun loadServerlessStackResources(): List<AwsExplorerNode<*>> {
+        try {
+            return cfnClient
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt]: pull the return out of the try and do `return try {`

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt [hunk 2]
```diff
@@ -105,23 +106,27 @@ class CloudFormationStackNode(project: Project, val stackName: String, private v
...
+
+                LambdaFunctionNode(
+                    nodeProject,
+                    client,
+                    response.configuration().toDataClass(credentialProvider.id, region),
+                    true
+                )
+            }
+            .toList()
+    } catch (e: Exception) {
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt]: When we add our 2nd service, we should make this more fine graned

**Pull request merged**